sys/shell/commands/shell_commands.c

```diff
@@ -279,6 +283,9 @@ const shell_command_t _shell_command_list[] = {
 #endif
 #ifdef MODULE_SUIT_COAP
     { "suit", "Trigger a SUIT firmware update", _suit_handler },
+#endif
+#ifdef MODULE_CRYPTOAUTHLIB
+    { "cryptoauth", "Commands for Microchip CryptoAuth devices", _cryptoauth },
```

Comment:
Ah, maybe I was too fast commenting beforehand. If you change to "CryptoAuth" here, the above help strings (`read`, `lock_c`, ...) should be updated too. However, I actually don't know what is clearer "CryptoAuth" or "atca" -> I leave it up to you but please align